quinn-h3/Cargo.toml

```diff
@@ -35,6 +35,7 @@ string = "0.2"
 tokio = "0.2.0-alpha.5"
 tokio-io = "0.2.0-alpha.5"
 tokio-codec = "0.2.0-alpha.5"
+webpki = "0.21"
```

Comment:
It doesn't look like this actually gets used?

Reply:
It's used here: https://github.com/djc/quinn/blob/adcd78aac74d21095cb96a7eed6cee74ce72d854/quinn-h3/src/client.rs#L63

Reply:
Ah, okay.